.devcontainer/devcontainer.json

```diff
@@ -26,8 +26,8 @@
 
 	// Invoke 'nvm' to install our preferred version of node, per the '.nvmrc' file
 	// located at the root of the ${workspaceFolder}.
-	// Also install pnpm as it is now required to install all packages
-	"postCreateCommand": ". /usr/local/share/nvm/nvm.sh; nvm install; npm i -g pnpm@7",
+	// Also run corepack enable to allow node to download and use the right version of pnpm.
```

Comment:
I don't see this running `corepack enable`. Does something else run that or is it missing?

Reply:
mistake; fixed.